src/devices/Mcp3428/ConversionResult.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Mcp3428::Mcp3428
```

Comment:
We require copyright header on all .cs files. I.e. https://github.com/dotnet/iot/blob/master/src/devices/Bno055/Bno055Sensor.cs#L1-L3

Reply:
Only missed one :)